src/OSPSuite.Presentation/Presenters/SimulationResidualVsTimeChartPresenter.cs

```diff
@@ -33,7 +33,7 @@ public class SimulationResidualVsTimeChartPresenter : SimulationVsObservedDataCh
       public SimulationResidualVsTimeChartPresenter(ISimulationVsObservedDataView view, ChartPresenterContext chartPresenterContext,
          IObservedDataRepository observedDataRepository, IResidualCalculatorFactory residualCalculatorFactory,
          IResidualsVsTimeChartService residualsVsTimeChartService)
-         : base(view, chartPresenterContext, ApplicationIcons.PredictedVsObservedAnalysis,
+         : base(view, chartPresenterContext, ApplicationIcons.ResidualVsTimeAnalysis,
```

Comment:
@georgeDaskalakis in MoBi, another presenter is used?
(just wondering because the icon in MoBi was displayed correctly before the fix)

Reply:
@Yuri05, yes in MoBi we have the fixed tabs, taht are explicitely named in code. The caption is the same one of course, coming from Core.